universal-login-wallet/src/ui/react/App.tsx

```diff
@@ -13,68 +13,71 @@ import {useServices} from '../hooks';
 import {WelcomeScreen} from './Home/WelcomeScreen';
 import {TermsAndConditionsScreen} from './Home/TermsAndConditionsScreen';
 import {CreateAccount} from './CreateAccount/CreateAccount';
-import {useModal} from '../hooks/useModal';
+import {createModalService, ModalContext} from '@universal-login/react';
+import {ModalStateType} from '../../core/entities/ModalStateType';
 
 const App = () => {
-  const modalService = useModal();
   const {walletService} = useServices();
+  const modalService = createModalService<ModalStateType>();
   const authorized = walletService.isAuthorized();
 
   return (
-    <Switch>
-      <Route
-        exact
-        path="/welcome"
-        render={() => <WelcomeScreen />}
-      />
-      <Route
-        exact
-        path="/terms"
-        render={() => <TermsAndConditionsScreen />}
-      />
-      <Route
-        exact
-        path="/create"
-        render={props => <CreateAccount {...props} modalService={modalService} />}
-      />
-      <Route
-        exact
-        path="/login"
-        render={props => <Login {...props} modalService={modalService} />}
-      />
-      <Route
-        exact
-        path="/approve"
-        render={() => <ApproveScreen />}
-      />
-      <Route
-        exact
-        path="/recovery"
-        render={() => <RecoveryScreen />}
-      />
-      <PrivateRoute
-        authorized={authorized}
-        exact
-        path="/"
-        render={() => <HomeScreen modalService={modalService} />}
-      />
-      <PrivateRoute
-        path="/transferring"
-        authorized={authorized}
-        render={() => <TransferringFundsScreen />}
-      />
-      <PrivateRoute
-        path="/notifications"
-        authorized={authorized}
-        render={() => <NotificationsScreen />}
-      />
-      <PrivateRoute
-        path="/settings"
-        authorized={authorized}
-        render={() => <SettingsScreen />}
-      />
-      <Route component={NotFound} />
-    </Switch>
+    <ModalContext.Provider value={modalService}>
```

Comment:
I think, that we should create `ModalContext` in the same place that we create `ServiceContext`, in [index.tsx](https://github.com/UniversalLogin/UniversalLoginSDK/blob/master/universal-login-wallet/src/index.tsx), because now it is inconsequential.